README.md

```diff
@@ -15,7 +15,7 @@ The syntax and available options are as follows:
 [
     {
         "alang": "jpn",
-        "slang": "eng",
+        "slang": ["en%-us", "eng"],
```

Comment:
Keep this as a string, the text below says that it can be an array, and there is an example below that shows the array format.

Reply:
Done